pages/firebase-outputs.js

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
+
+import {
+  getListOfAllArticlesByAlphabet,
+  getListOfAllArticlesByDate,
+  getListOfLastArticles,
+} from '../firebase/api/articles';
+
+const FirebaseFirst = () => {
+  const [articleList, setArticleList] = useState([]);
+  const [latestArticles, setLatestArticles] = useState([]);
+  const [alphabeticalListOfArticles, setAlphabeticalListOfArticles] = useState(
+    [],
+  );
+
+  useEffect(() => {
+    const listOfArticles = getListOfAllArticlesByDate();
+    listOfArticles.then((value) => setArticleList(value));
+
+    const listOfLastestArticles = getListOfLastArticles();
+    listOfLastestArticles.then((value) => setLatestArticles(value));
+
+    const listOfArticlesAlphabet = getListOfAllArticlesByAlphabet();
+    listOfArticlesAlphabet.then((value) =>
+      setAlphabeticalListOfArticles(value),
+    );
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div className="homepage">
+      <div>
+        <h2 className="headline--1 mb-4">All articles</h2>
```

Comment:
Have we switched to english? :D

Reply:
This is only helper/temporary file and many things inside is weird :D.